### PR TITLE
Added cnvkit for tumor_only

### PR DESCRIPTION
**Pull request by SusiJo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by SusiJo** [comment]
Note: the module `cnvkit/batch` needs to be adjusted and needs to test if normal samples are provided. 
Also, the new local modules should probably be implemented in `nf-core/modules` and should be removed here later.

File: CHANGELOG.md [hunk 1]
```diff
@@ -73,7 +74,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#513](https://github.com/nf-core/sarek/pull/513), [#527](https://github.com/nf-core/sarek/pull/527) - CNV is back
 - [#529](https://github.com/nf-core/sarek/pull/529) - Do not save `versions.yml` files
 - [#524](https://github.com/nf-core/sarek/pull/524) - Fix intervals usage by counting the actual list of scatter/gather files produced and not overall number of intervals
-- [#549](https://github.com/nf-core/sarek/pull/549) - Fix unique lanes required for Freebayes: issue [#311](https://github.com/nf-core/sarek/issues/311), replaces `meta.clone()` with actual copy of map to avoid issues with https://nfcore.slack.com/archives/C027CM7P08M/p1644241819942339
+- [#549](https://github.com/nf-core/sarek/pull/549) - Fix unique lanes required for Freebayes: issue [#311](https://github.com/nf-core/sarek/issues/311), replaces `meta.clone()` with actual copy of map to avoid issues with <https://nfcore.slack.com/archives/C027CM7P08M/p1644241819942339>
```
Comment (FriederikeHanssen) [inline CHANGELOG.md:77]: bit confused this is part of the PR 🤔

File: conf/modules.config [hunk 2]
```diff
@@ -904,6 +904,39 @@ process{
...
+      withName: 'CNVKIT_ANTITARGET' {
+        ext.when         = { params.tools && params.tools.contains('cnvkit') }
+        publishDir       = [
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/reference" },
+            pattern: "*{bed}"
+        ]
+    }
+
+
```
Comment (FriederikeHanssen) [inline conf/modules.config:918]:
```suggestion
```

File: subworkflows/local/tumor_variant_calling.nf [hunk 3]
```diff
@@ -83,6 +84,20 @@ workflow TUMOR_ONLY_VARIANT_CALLING {
         ch_versions = ch_versions.mix(RUN_CONTROLFREEC_TUMORONLY.out.versions)
     }
 
+    if(tools.contains('cnvkit')){
+        cram_recalibrated_cnvkit = cram_recalibrated
```
Comment (FriederikeHanssen) [inline subworkflows/local/tumor_variant_calling.nf:88]: same here about the `crai`

**Review by FriederikeHanssen — COMMENTED** [review]
Looks really good already. Just want to clear up the `crai` thing before merging

File: CHANGELOG.md (hunk 1 above)
Comment (FriederikeHanssen) [reply CHANGELOG.md:77]: just prettier being prettier

File: subworkflows/local/tumor_variant_calling.nf (hunk 3 above)
Comment (FriederikeHanssen) [reply subworkflows/local/tumor_variant_calling.nf:88]: all good, samtools computes the bai index, so this is all fine

**Review by FriederikeHanssen — APPROVED** [review]
🚀

**Pull request merged**